README.md

```diff
@@ -25,9 +25,7 @@ Mifos boasts an active and vibrant contributor community, Please join us on [sla
 
 ## How to Contribute
 
-Thank you for your interest in contributing to the Mobile Wallet project by Mifos! We welcome all contributions and encourage you to follow these guidelines to ensure a smooth and efficient collaboration process.
-
-The issues should be raised via the GitHub issue tracker. For Issue tracker guidelines please click <a href="https://github.com/openMF/mobile-wallet/blob/master/.github/CONTRIBUTING.md#issue-tracker">here</a>. All fixes should be proposed via pull requests. For pull request guidelines please click <a href="https://github.com/openMF/mobile-wallet/blob/master/.github/CONTRIBUTING.md#pull-requests">here</a>. For commit style guidelines please click <a href="https://github.com/openMF/mobile-wallet/wiki/Commit-style-guide">here</a>.
+Click here to (https://github.com/openMF/mobile-wallet/wiki/How-to-Contribute)
```

Comment:
Remove this and make it `How to Contribute` as link like this 

### [How to Contribute](https://github.com/openMF/mobile-wallet/wiki/How-to-Contribute)